docs/Wdio_Utility.md

```diff
@@ -62,76 +61,54 @@ There are a few things to note about the webdriver.io configuration provided by
+- `Terra.it.validatesElement()` mocha-chai convenience method that takes a screenshot and verifies the images are within the specified mis-match tolerance and performs accessibility validation. Note: this method provides its own mocha it test case. Also, since viewports isn't accepted in this method, you need to [test in multiple viewports](https://github.com/cerner/terra-toolkit/blob/master/tests/wdio/describeViewports-spec.js). This method accepts these arguments (in this order):
+    - String (optional): the test case name. Default name is 'default'
+    - Object (optional): the test options. Options include selector, misMatchTolerance and axeRules:
+         - selector: the element selector to take a screenshot of. Defaults to the global terra.selector.
+         - misMatchTolerance: number between 0 and 100 that defines the degree of mismatch to consider two images as identical, increasing this value will decrease test coverage. Defaults to the global visualRegression.compare.misMatchTolerance.
+         - axeRules: the axe rules to use as overrides if necessary.
+    - See [validateElement-spec.js](https://github.com/cerner/terra-toolkit/blob/master/tests/wdio/validateElement-spec.js) for example usage.
+    - This helper can be used inside mocha `it` blocks by being called through `Terra.validates.element()`. This takes the same arguments.
+- `Terra.it.isAccessible()` mocha-chai convenience method that runs an axe test for the page.
     - See [beAccessible-spec.js](https://github.com/cerner/terra-toolkit/blob/master/tests/wdio/beAccessible-spec.js) for examples.
-- `Terra.should.matchScreenshot()` **Note - It is preferred to use Terra.should.validateElement().  Terra.should.matchScreenshot() may eventually be deprecated** mocha-chai convenience method that takes a screenshot for the specified viewports and verifies the images are within the specified mis-match tolerance. Note: this method provides its own mocha it test case. The methods accepts these arguments (in this order):
+    - This helper can be used inside mocha `it` blocks by being called through `Terra.validates.accessibliity()`. This takes the same arguments.
+- `Terra.it.matchesScreenshot()` **Note - It is preferred to use Terra.it.validatesElement().  Terra.should.matchScreenshot() may eventually be deprecated** mocha-chai convenience method that takes a screenshot for the specified viewports and verifies the images are within the specified mis-match tolerance. Note: this method provides its own mocha it test case. The methods accepts these arguments (in this order):
     - String (optional): the test case name. Default name is 'default'
     - Object (optional): the test options. Options include selector, viewports, and misMatchTolerance:
          - selector: the element selector to take a screenshot of. Defaults to the global terra.selector.
          - viewports: the array of viewports dimensions to take a screenshot in. Defaults to the current viewport size.
```

Comment:
Can you add a note saying this shouldn't be used as it can significantly slow down test performance and can caused screenshot inconsistencies with some responsive UI

Reply:
Done in 04987d77e2d549eee0d8c03ad3f07fe0fa4103c2